src/main/java/mekanism/common/tile/component/SideConfig.java

```diff
@@ -0,0 +1,56 @@
+package mekanism.common.tile.component;
+
+import net.minecraft.util.EnumFacing;
+
+import java.util.Observable;
+import java.util.Observer;
+
+public class SideConfig {
+    private byte[] directions = new byte[EnumFacing.VALUES.length];
+    private Observable observable = new Observable();
```

Comment:
Shouldn't `SideConfig` extend `Observable`?

Also appears `Observable#setChanged` should be called before `notifyObservers`